### PR TITLE
DBZ-2690 get oracle specific changes into debezium-core

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-2690

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java [hunk 1]
```diff
@@ -1202,6 +1202,8 @@ public void readSchema(Tables tables, String databaseCatalog, String schemaNameP
      * included in column.include.list.
      */
     protected Optional<ColumnEditor> readTableColumn(ResultSet columnMetadata, TableId tableId, ColumnNameFilter columnFilter) throws SQLException {
+        final String defaultValue = columnMetadata.getString(13);
```
Comment (jpechane) [inline debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java:1205]: @ani-sha Could you please just add a comment explaining that this is Oracle specific so it must stay there?

Comment (jpechane) [reply debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java:1205]: See `io.debezium.connector.oracle.OracleConnection.readTableColumn(ResultSet, TableId, ColumnNameFilter)`. Also there should be an incubator rep pr that would remove this method as it should be same as in the core

Comment (ani-sha) [reply debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java:1205]: Sure thing. Will add them.

**Comment by jpechane** [comment]
@ani-sha Applied, thanks!

**Pull request merged**